src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -512,12 +512,13 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		)
 		
 		let footerViewModel = FooterViewModel(
-			primaryButtonName: AppStrings.ExposureSubmissionTestResultAvailable.primaryButtonTitle,
-			primaryIdentifier: AccessibilityIdentifiers.ExposureSubmissionTestResultAvailable.primaryButton,
-			isSecondaryButtonEnabled: false,
-			isSecondaryButtonHidden: true
+			primaryButtonName: AppStrings.SRSConsentScreen.primaryButtonTitle,
+			secondaryButtonName: AppStrings.SRSConsentScreen.secondaryButtonTitle,
+			primaryIdentifier: AccessibilityIdentifiers.SRSConsentScreen.primaryButton,
```

Comment:
Please attend to UI Test `test_SRSFlow_PositiveWithoutResultInTheApp()`: Identifier:

```swift
app.buttons[AccessibilityIdentifiers.ExposureSubmissionTestResultAvailable.primaryButton].waitAndTap(.long)
```

Reply:
I will fix this